commons/src/types/Nylas.ts

```diff
@@ -317,6 +317,7 @@ export interface Thread {
   folder_id?: string;
   label_ids?: string[];
   selected?: boolean;
+  drafts?: Message[];
```

Comment:
Should the `body` prop in the `Message` interface be changed to optional if the drafts array is typed this way?